homeassistant/components/proliphix/climate.py

```diff
@@ -61,11 +61,6 @@ def __init__(self, pdp):
         self._pdp = pdp
         self._name = None
 
-    @property
-    def should_poll(self):
-        """Set up polling needed for thermostat."""
-        return True
-
```

Comment:
True is the default return value for Entity (and it is not overriden in ClimateEntity)